manual/src/cmds/tail-mod-cons.etex

```diff
@@ -150,6 +158,31 @@ tail-mod-cons are as follows:
   using a convenient monadic notation.
 \end{itemize}
 
+\paragraph{Note: OCaml call stack size} Under OCaml 4, bytecode
```

Comment:
`Under OCaml 4` seems unclear to me, I would propose `With OCaml versions anterior to OCaml 5`.